account_tax_legal_notes_translate/models/account_tax.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    invoice_legal_notes_translate = fields.Html(
```

Comment:
It's a backport, so use the same name as Odoo in order to avoid migration scripts:
https://github.com/odoo/odoo/blob/19.0/addons/account/models/account_tax.py#L232


```suggestion
    invoice_legal_notes = fields.Html(
```